manifests/l/LGUG2Z/whkd/0.1.1/LGUG2Z.whkd.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.7-3-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: LGUG2Z.whkd
+PackageVersion: 0.1.1
+MinimumOSVersion: 10.0.0.0
+InstallerLocale: en-US
+InstallerType: wix
+Scope: machine
+UpgradeBehavior: install
+Commands:
+- whkd
+ReleaseDate: 2023-02-14
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/LGUG2Z/whkd/releases/download/v0.1.1/whkd-0.1.1-x86_64.msi
+  InstallerSha256: 1dc9798953d1b3eff35db4fba73e1bb074e7ffb8e0e9b224206c340d78945b51
+  ProductCode: '{EB940FAD-0B97-47AE-A3ED-1F2F5972B8D8}'
```

```suggestion
  ProductCode: '{F0EDC6C1-CA34-4BC7-B34A-8476D625DC4E}'
```
This must match Windows Registry.